File: src/main/java/com/rest/app/makePDF/ComparisonPDF.java
package com.rest.app.makePDF;

import com.testautomationguru.utility.PDFUtil; // добавляем библиотеку для работы с PDF
import com.rest.app.dataBaseK.MyPdfFiles;

public class ComparisonPDF {

    public void compPDF() throws Exception {   // сравнение
        PDFUtil pdfUtil = new PDFUtil();      // читает PDF и переводит его в String
        boolean comparison = pdfUtil.compare(MyPdfFiles.INSTANCE.getMyReport(), MyPdfFiles.INSTANCE.getMyReport2()); // INSTANCE дает ссылку на объект "object класса MyPdfFiles
        System.out.println("compPDF: myReport vs myReport2 = " + comparison);
    }

    public void compPDF(String a, String b) throws Exception {
        PDFUtil pdfUtil = new PDFUtil();      // читает PDF и переводит его в String
        boolean comparison = pdfUtil.compare(a, b);
        System.out.println("compPDF(a, b): " + a + " vs " + b + " = " + comparison);
    }
}

File: src/main/java/employees/makePDF/ComparisonPDF.kt
package employees.makePDF

import employees.database.MyPdfFiles.getMyReport
import employees.database.MyPdfFiles.getMyReport2
import com.testautomationguru.utility.PDFUtil

class ComparisonPDF {
    // сравнение файлов
    @Throws(Exception::class)
    fun compPDF() {
        val pdfUtil = PDFUtil() // читает PDF и переводит его в String
        val comparison = pdfUtil.compare(getMyReport(), getMyReport2()) // MyPdfFiles.INSTANCE.
        println("compPDF: myReport vs myReport2 = $comparison")
    }

    // сравнение файлов на выбор
    @Throws(Exception::class)
    fun compPDF(a: String, b: String) {
        val pdfUtil = PDFUtil() // читает PDF и переводит его в String
        val comparison = pdfUtil.compare(a, b)
        println("compPDF(a, b): $a vs $b = $comparison")
    }
}
